projects/burnr/src/Home.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from "react"
+import * as React from "react"
```

Comment:
For consistency with the rest of the code base, could we please change this to?

```ts
import { useState, useEffect } from "react"
```

Reply:
This gives an error:
```
'React' refers to a UMD global, but the current file is a module. Consider adding an import instead
```